table.csv

```diff
@@ -483,6 +483,12 @@ ssz,                            serialization,  0xb501,         draft,     Simpl
 ssz-sha2-256-bmt,               multihash,      0xb502,         draft,     SSZ Merkle tree root using SHA2-256 as the hashing function and SSZ serialization for the block binary
 iscc,                           softhash,       0xcc01,         draft,     ISCC (International Standard Content Code) - similarity preserving hash
 zeroxcert-imprint-256,          zeroxcert,      0xce11,         draft,     0xcert Asset Imprint (root hash)
+varsig,                         varsig,         0xd000,         draft,     Namespace for all not yet standard signature algorithms
```

Comment:
I'm surprised you're not going to make a claim for a single-byte range for this, is that because these signatures are all going to be fairly large anyway so an extra byte isn't really a big deal?
Certainly makes this easier!

Reply:
A single byte range is probably too small if we want a consistent first digit. There are definitely more than 16 signature types out there. We're already at 10 in this PR, plus there's an additional 2 for Holochain that are not in this range (but are in the table).

Reply:
And indeed, we thought about compressing these into a tightly packed list. We definitely will need more than 16 (to accommodate signatures like EIP-191, possibly passkeys, and other nonstandard signature types).

I'm not completely against just counting up from `0xd000`, but the current layout is very easy to understand in relation to the existing PKs codes. As you say: relative to the size of a signature, we likely won't notice the extra byte in the 3-byte section.

Reply:
Oh, so I misunderstood the intention of this one, I think. You're not really using it as a signal value to say "here follows a varsig", but a catch-all of some kind. Is the intent here that you're also making a claim on bytes starting with `0xd000` to `0xdfff` and we should be careful about allocating those in the future?

Reply:
@expede I think you've misunderstood @rvagg comment, which if I'm understanding correctly simply asks why not pic a different, small code so it takes 1 byte as opposed to 3 used necessary for `0xdfff`.

@rvagg you are correct that this code is sort of catch-all and specifically it has been [defined as follows](https://github.com/ucan-wg/ucan-ipld/#251-nonstandard-signatures)

> ### 2.5.1 Nonstandard Signatures
>
> Nonstandard signature algorithms MAY be used, but are NOT RECOMMENDED as they are not widely interoperable.
>
> The `0xd000` prefix MUST be used if a nonstandard signature is used. Nonstandard signatures MUST be additionally prefixed with a varint of the signature length, and the UTF-8 UCAN `alg` field appended afterwards.
>
>```
> <varint 0xd000><varint sig_size><bytes sig><utf8 alg>`
> ```


I do not have strong preference between short and more memorable codes, but if I had to choose I'd probably side with former as these aren't meant to be memorized.